src/components/button-toggle/button-toggle.style.ts

```diff
@@ -43,7 +43,7 @@ const StyledButtonToggleContentWrapper = styled.div`
 `;
 
 export interface StyledButtonToggleLabelProps {
-  /** buttonIcon to render. */
+  /** Button icon to render. */
```

Comment:
If you're fixing this it might be better to say something like `Icon that is rendered inside of the button`. Doesn't have to be that exactly but think of something you'd like to read if you'd never used Carbon before.